releasenotes/notes/npm-udpv6-receive-1a62cf8b6080b7c6.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Add missing support for UDPv6 receive path to NPM
```

Comment:
```suggestion
    Add missing support for UDPv6 receive path to NPM.
```

Reply:
Sorry this was mistakenly opened against the `main` branch instead of the `npm-main` branch; this diff shouldn't have been there.